Analyzer/interface/CommonFunction.h

```diff
@@ -1505,23 +1505,17 @@ reco::DeDxData computedEdx(const int& run_number,
                            float* scaleFactors,
                            TH3* templateHisto = nullptr,
                            bool usePixel = false,
+                           bool useStrip = true,
                            bool useClusterCleaning = true,
-                           bool reverseProb = false,
                            bool useTruncated = false,
-                           std::unordered_map<unsigned int, float>* TrackerGains = nullptr,
-                           bool useStrip = true,
                            bool mustBeInside = false,
                            size_t MaxStripNOM = 999,
                            bool correctFEDSat = false,
                            int crossTalkInvAlgo = 0,
                            float dropLowerDeDxValue = 0.0,
-                           dedxHIPEmulator* hipEmulator = nullptr,
                            float* dEdxErr = nullptr,
-                           unsigned int pdgId = 0,
-                           bool skipPixel = true,
                            bool useTemplateLayer = false,
                            bool skipPixelL1 = false,
-                           bool probQ = false,
```

Comment:
we dont need this anymore

Reply:
We need this for a certain computation of ProbQ.

Reply:
But that seemed to have been a misunderstanding, that Ias definition has nothing to do with probQ, it's just combining the charges with that log method... (which happens to be used in the probQ too, but really that's just a coincidence)

Reply:
Yes I know it's not the 'real' ProbQ, it was just another way to combine the probabilities used in Ias.

Reply:
But that would be applied on probabilities... not on the charges. Anyway, did you look into how that Ias behaved? Can we just get rid of it after all?

Reply:
"But that would be applied on probabilities... not on the charges"
Yes and its what's done, it was the purpose. 
I'm still investigating the Ias behavior.